### ISSUE TITLE
llama-parse install not working due to version compatibility issue w/ llama-index

### ISSUE THREAD

**Issue by jbbae** [run-llama/llama_parse#112]
I'm using poetry to manage my dependencies. I already have the latest version of llama-index installed (0.10,20).

When I try "poetry add llama-parse", I get the below error:

```
Because no versions of llama-index-readers-llama-parse match >0.1.2,<0.1.3 || >0.1.3,<0.2.0
 and llama-index-readers-llama-parse (0.1.2) depends on llama-parse (>=0.3.3,<0.4.0), llama-index-readers-llama-parse (>=0.1.2,<0.1.3 || >0.1.3,<0.2.0) requires llama-parse (>=0.3.3,<0.4.0).
And because llama-index-readers-llama-parse (0.1.3) depends on llama-parse (>=0.3.3,<0.4.0), llama-index-readers-llama-parse (>=0.1.2,<0.2.0) requires llama-parse (>=0.3.3,<0.4.0).
Because no versions of llama-index match >0.10.20,<0.11.0
 and llama-index (0.10.20) depends on llama-index-readers-llama-parse (>=0.1.2,<0.2.0), llama-index (>=0.10.20,<0.11.0) requires llama-index-readers-llama-parse (>=0.1.2,<0.2.0).
Thus, llama-index (>=0.10.20,<0.11.0) requires llama-parse (>=0.3.3,<0.4.0).
So, because app depends on both llama-index (^0.10.20) and llama-parse (^0.4.0), version solving failed.
```

How do I resolve this to install llama-parse while still using poetry...?

I would try changing the version trying to be installed, but per your GitHub there are no versions for llama_parse...?

**Comment by MeTaNoV** [comment]
should be fixed quickly! @logan-markewich ;)

**Comment by MeTaNoV** [comment]
@jbbae, I think it is fixed now! Managed to install a 0.4.0 in one of my Colab!

**Comment by logan-markewich** [comment]
Yes this should be fixed now
